docs/source/conf.py

```diff
@@ -50,7 +50,8 @@
     'sphinx.ext.todo',  # enable to-do sections
     'sphinx.ext.graphviz',  # enable graphs for breathe
     'sphinx.ext.viewcode',  # add links to view source code
-    'sphinx_copybutton',  # add a copy button to code blocks
+    'sphinx_copybutton',  # add a copy button to code blocks,
+    'sphinx_inline_tabs' # add tabs
```

Comment:
Needs two spaces before `#` to satisfy linter... removed extra comma after comment... added other comment at end of list (I know it doesn't do anything other than prevent lines changing when you add new extensions).

```suggestion
    'sphinx_copybutton',  # add a copy button to code blocks
    'sphinx_inline_tabs',  # add tabs
```